src/components/shared/DateAndDebit/DateAndDebit.module.scss

```diff
@@ -0,0 +1,15 @@
+@import "styles/theme";
+
+.layoutDateAndDebit {
+  display: flex;
```

Comment:
Consider creating layout styles for flex style sets inside your theme. You can implement these as classes or mixins that your CSS can use.